ghcide/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -24,6 +24,9 @@ import           Control.Arrow                                     (second,
 import           Control.Concurrent.STM.Stats                      (atomically)
 import           Control.Monad                                     (guard, join)
 import           Control.Monad.IO.Class
+#if !MIN_VERSION_ghc(9,0,0)
+import           Ctype                                             (is_ident)
+#endif
```

Comment:
Please move all the CPP to the compatibility layer at `Development.IDE.Compat.GHC`

Reply:
Thank you for the pointing that out. Fixed.